x-pack/plugins/reporting/server/usage/types.ts

```diff
@@ -117,8 +119,7 @@ export type ReportingUsageType = RangeStats & {
   last7Days: RangeStats;
 };
 
-export type ExportType = 'csv' | 'csv_searchsource' | 'printable_pdf' | 'PNG';
-export type FeatureAvailabilityMap = { [F in ExportType]: boolean };
+export type FeatureAvailabilityMap = Record<string, boolean>;
```

Comment:
Changed this to be indexed by `string` instead `ExportType` for easier maintenance. 

Instead of keeping a hardcoded list of all the known export types, we can use `exportTypesHandler.getJobTypes()`: it returns the jobTypes of all registered export types.